app/models/concerns/audit_associations.rb

```diff
@@ -0,0 +1,59 @@
+module AuditAssociations
+  def self.included(base)
```

Comment:
@ares How do you suggest to eliminate the `self.included` methods from these modules. Any better, idiomatic ruby way to do it?

Reply:
ActiveSupport::Concern has support for something like
```ruby
included do
  ...
end
```

but given how many headaches it caused when we had to change alias_method_chain and start using prepend I think pure Ruby callback is good. So I'd fine with this approach.